airbyte-local.sh

```diff
@@ -459,14 +738,73 @@ function sync() {
     cp "$new_source_state_file" "$src_state_filepath"
 }
 
-function cleanup() {
-    if [[ -s "$tempPrefix-src_cid" ]]; then
-        docker container kill $(cat "$tempPrefix-src_cid") 2>/dev/null || true
-        rm "$tempPrefix-src_cid"
+function sync_kube() {
+    local kube_manifest_template=connection-pod.yaml
+    kube_manifest_tmp=__${kube_manifest_template}
+    pod_name=$(generatePodName)
+    local pod=$pod_name
+    local namespace=$kube_namespace
+    local src_container=source
+    local dst_container=destination
+    local state_container=transformer
+    local new_state_file=new_state.json
+
+    generateKubeManifest ${kube_manifest_tmp}
+    kubectl apply -f ${kube_manifest_tmp} -n ${namespace}
+    # Wait for init container to start up
+    waitForPodContainer $namespace $pod init "running"
+    echo "Copying config files to pod $pod"
+    kubectl cp $tempdir/$src_config_filename $pod:/config/source_config.json -n ${namespace} -c init
+    if [[ -s "$tempdir/$src_catalog_filename" ]]; then
+        kubectl cp $tempdir/$src_catalog_filename $pod:/config/source_catalog.json -n ${namespace} -c init
+    fi
+    kubectl cp $tempdir/$dst_config_filename $pod:/config/destination_config.json -n ${namespace} -c init
+    if [[ -s "$tempdir/$dst_catalog_filename" ]]; then
+        kubectl cp $tempdir/$dst_catalog_filename $pod:/config/destination_catalog.json -n ${namespace} -c init
+    fi
+    kubectl cp $tempdir/$src_state_filename $pod:/config/input_state.json -n ${namespace} -c init
+    touch $tempdir/FINISHED_UPLOADING && kubectl cp $tempdir/FINISHED_UPLOADING $pod:/config/ -n ${namespace} -c init
+
+    # Wait for source container to start up
+    waitForPodContainer $namespace $pod $src_container "running"
+    # Tail source logs in the background
+    log "Tailing source container logs in the background"
+    kubectl logs -f $pod -c $src_container -n ${namespace} | jq -cR $jq_color_opt --unbuffered 'fromjson?' | jq -rR "$jq_src_msg" &
+
+    waitForPodContainer $namespace $pod $dst_container "running"
+    log "Tailing destination container logs in the background"
+    kubectl logs -f $pod -c $dst_container -n ${namespace} | JQ_COLORS="1;30:0;37:0;37:0;37:0;36:1;37:1;37" jq -cR $jq_color_opt --unbuffered 'fromjson?' | jq -rR "$jq_dst_msg"
```

Comment:
do we even care about the colors?

Reply:
using the same output/colors format as CLI